### PR TITLE
Add generic JS wrapClient method

### PR DESCRIPTION
**Pull request by jacoblee93**
CC @dqbd @hinthornw

### CODE REVIEW

File: js/src/client.ts [hunk 1]
```diff
@@ -853,6 +853,7 @@ export class Client {
         signal: AbortSignal.timeout(this.timeout_ms),
       }
     );
+    console.log(response);
```
Comment (dqbd) [inline js/src/client.ts:856]: remove

**Comment by jacoblee93** [comment]
Example trace: https://smith.langchain.com/public/323d9897-2014-44eb-99d1-aa22691fa34a/r

File: js/src/client.ts (hunk 1 above)
Comment (jacoblee93) [reply js/src/client.ts]: Thanks!

**Comment by jacoblee93** [comment]
We might have to add an Anthropic key to CI if we want to do this @hinthornw

**Review by dqbd — APPROVED** [review]

**Review by hinthornw — APPROVED** [review]
Do we need some custom handling to show how to handle streaming or other outputs (make them look like OAI format)?

**Pull request merged**